pages/learn/course/games-and-demos.vue

```diff
@@ -69,6 +69,24 @@ export default class GamesAndDemosPage extends CourseOverviewPage {
       label: 'Widgets Demonstration',
       segment: { cta: 'widgets-demonstrationr', location: 'course' },
       url: GamesAndDemos.WidgetsDemonstration
+    },
+    {
+      image: 'coin-game.png',
+      label: 'Quantum Coin Game',
+      segment: { cta: 'coin-game', location: 'course' },
+      url: GamesAndDemos.CoinGame
+    },
+    {
+      image: 'first-quantum-game.png',
+      label: 'Your First Quantum Game',
```

Comment:
now I see why you're using "Your First Quantum Game" because this is what the page title is... 

I'm fine with leaving dropdown menu as "Your First Quantum Game", then... thanks for catching this. (We should open a PR in the Platypus / Textbook beta repo, to address this there)